ambari-agent/src/main/python/ambari_agent/Hardware.py

```diff
@@ -40,7 +40,7 @@ class Hardware:
   CHECK_REMOTE_MOUNTS_KEY = 'agent.check.remote.mounts'
   CHECK_REMOTE_MOUNTS_TIMEOUT_KEY = 'agent.check.mounts.timeout'
   CHECK_REMOTE_MOUNTS_TIMEOUT_DEFAULT = '10'
-  IGNORE_ROOT_MOUNTS = ["proc", "dev", "sys"]
+  IGNORE_ROOT_MOUNTS = ["proc", "dev", "sys", "boot"]
```

Comment:
I'd also add /run if relevant

Reply:
while there were no problems with /run, it is nice idea to include it to ignore list

Reply:
oh sorry, re-based again after fix =\ .... no need to add run as we have filter based on file-system types. By default we filtering tempfs mounts, and run is exactly tempfs volume